cmd/security-agent/main_nix.go

```diff
@@ -14,13 +14,16 @@ import (
 	_ "expvar"         // Blank import used because this isn't directly used in this file
 	_ "net/http/pprof" // Blank import used because this isn't directly used in this file
 
+	"github.com/DataDog/datadog-agent/cmd/internal/rssshrinker"
 	"github.com/DataDog/datadog-agent/cmd/internal/runcmd"
 	"github.com/DataDog/datadog-agent/cmd/security-agent/command"
 	"github.com/DataDog/datadog-agent/cmd/security-agent/subcommands"
 	"github.com/DataDog/datadog-agent/pkg/util/flavor"
 )
 
 func main() {
+	rssshrinker.Setup()
```

Comment:
I think this should go under a config flag, and be disabled by default at first while we enable this slowly internally. I don't understand how it could be possible that this does not impact the CPU usage of the agent(s)

Reply:
We've been discussing this on Slack but this was also my thinking (both for the config flag and the cpu usage)